### PR TITLE
SUBMARINE-498. [WEB] Add notebook module to workbench with Angular

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Add notebook module into workbench with angular by importing ngx-monaco-editor.
Currently is functional presentation.

### What type of PR is it?
[Feature]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-498

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine

### Screenshots (if appropriate)
notebookDemo

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@liuxunorg @jasoonn @pingsutw @chiajoukuo Please help me preview this pr. Thanks a lot.

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/notebookIT.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    driver =  WebDriverManager.getWebDriver();
+  }
+
+  @AfterClass
+  public static void tearDown(){
+    driver.quit();
+  }
+
+  @Test
+  public void workspaceNavigation() throws Exception {
```
Comment (jasoonn) [inline submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/notebookIT.java:47]: The title of the function would be more suitable if it is changed to notebookNavagation.

**Comment by chiajoukuo** [comment]
@kobe860219 Thank for your contribution. This pr looks good to me!

File: submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/notebookIT.java (hunk 1 above)
Comment (kobe860219) [reply submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/notebookIT.java]: Thanks ! I have renamed it.

**Comment by liuxunorg** [comment]
@kobe860219 coooooool !!!
This is important feature!
@lleohao submitted a notebook PR before [SUBMARINE-307. Add notebook module](https://github.com/apache/submarine/pull/256), but because he was busy, he did not continue to complete it. You can refer to his PR implementation to improve it.

**Comment by kobe860219** [comment]
@liuxunorg Thanks ! I will refer it to improve and develope.

**Comment by liuxunorg** [comment]
@kobe860219 Please rebase this commit.

**Comment by kobe860219** [comment]
@liuxunorg Thanks a lot ! I have updated !

**Comment by lleohao** [comment]
Thanks

**Pull request closed without merging**